general/sonar-integration.md

```diff
@@ -0,0 +1,37 @@
+# Sonarcloud
+
```

Comment:
please remove the empty lines between headers and following paragraph.

Reply:
Done